deploy/helm/opa-operator/templates/deployment.yaml

```diff
@@ -14,11 +14,12 @@ spec:
       {{- include "operator.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
       annotations:
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+        {{- with .Values.podAnnotations }}
```

Comment:
Move into templating

Reply:
Is the `image` attribute special, or should we be looking at prefixing it, eg: `stackable.tech/image`?

Reply:
It's arbitrary, that's probably a good call.

Reply:
381ac91a56f91d8806f05ecf405c2d1862a00ac2

Reply:
@nightkr, should it be added into the [deployment template](https://github.com/stackabletech/operator-templating/blob/2b73b32beb75a63600e50a962c50b473fda62021/template/deploy/helm/%5B%5Boperator%5D%5D/templates/deployment.yaml.j2#L7-L8) directly, or into [the helper](https://github.com/stackabletech/operator-templating/blob/2ef5a917cce8d1710e1ce0d099947663f74b4428/template/deploy/helm/%5B%5Boperator%5D%5D/templates/_helpers.tpl#L40-L50)?

Reply:
Or the [selectorLabels](https://github.com/stackabletech/operator-templating/blob/2ef5a917cce8d1710e1ce0d099947663f74b4428/template/deploy/helm/%5B%5Boperator%5D%5D/templates/_helpers.tpl#L52-L58)

Reply:
My assumption is the deployment directly, because there are references to the annotations as mentioned in the comments below...

Reply:
I will also mention that `.Values.image.tag` is not defined: 

https://github.com/stackabletech/opa-operator/blob/b1644e4e49fd84879442fc9333cc947147faf47f/deploy/helm/opa-operator/values.yaml#L3-L6

Reply:
Done in https://github.com/stackabletech/operator-templating/pull/294